tests/BinaryFormatterTests/WhenWorkingWith_Classes.cs

```diff
@@ -55,5 +146,31 @@ public void CanWorkWith_ClasseWithCustomCtor_WithProperties_WithPublicSetter()
             Assert.Equal(before.Int, after.Int);
             Assert.Equal(before.Double, after.Double);
         }
+
+        [Fact]
+        public void CanWorkWith_ClassesWithoutCustomCtor_WithVirtualProperties_WithPublicSetter()
```

Comment:
It's difficult to understand whats happening in this test. 
WhenWorkingWith_Classes ctor adds an element to the hidden static list. Looks like side-effect
Can you write it more explicitly?